operational/azure/azure_migrate/README.md

```diff
@@ -25,6 +25,21 @@ This policy has the following input parameters required when launching the polic
 
 - Populate an Azure Migrate project with the discovered servers from a RISC Foundations Assessment
 
+## Prerequisites
```

Comment:
Should this be combined with the previous 'Pre-requisites' section or should that be renamed?

Reply:
good catch.  I moved the new items to the existing list.

Reply:
We could probably also get rid of the dash.

Reply:
removed the dash.